ui/jstests/test-learning-resource.jsx

```diff
@@ -119,76 +128,117 @@ define(['QUnit', 'jquery', 'react', 'lodash', 'learning_resources',
+    }
+  );
+
+  QUnit.test(
+    'Assert that TermSelect renders properly',
+    function (assert) {
+      var done = assert.async();
+
+      var afterMount = function(component) {
+        var $node = $(React.findDOMNode(component));
+
+        var $termSelect = $node.find("select");
+        assert.equal($termSelect.size(), 1);
 
         done();
       };
 
       React.addons.TestUtils.renderIntoDocument(
-        <VocabularyOption vocabulary={vocabularyResponseDifficulty}
-                          terms={difficultyTermsResponse.results}
-                          selectedTerm="hard"
-                          ref={afterMount}/>
+        <TermSelect
+          vocabs={vocabulariesAndTerms}
+          selectedVocabulary={selectedVocabulary}
+          ref={afterMount}
+        />
       );
     }
   );
 
   QUnit.test(
+    'Assert that TermList renders properly',
+    function (assert) {
+      var done = assert.async();
+      var afterMount = function(component) {
+        var $node = $(React.findDOMNode(component));
+        var $termList = $node.find("ul");
+        assert.equal($termList.size(), 1);
+        done();
+      };
+
+      React.addons.TestUtils.renderIntoDocument(
+        <TermList
+          vocabs={vocabulariesAndTerms}
+          ref={afterMount}
+        />
+      );
+    }
+  );
+  QUnit.test(
     'Assert that LearningResourcePanel changes state properly',
     function(assert) {
       var done = assert.async();
-
       var afterMount = function(component) {
         // wait for calls to populate form
         waitForAjax(3, function () {
-          // one vocabulary
+          // two menus: vocabulary and terms.
           var $node = $(React.findDOMNode(component));
-          var $vocabSelect = $node.find("select");
-          assert.equal($vocabSelect.size(), 2);
 
-          // two terms, first vocab
-          var $terms1Select = $($vocabSelect[0]).find("option");
-          assert.equal($terms1Select.size(), 2);
-          assert.equal($terms1Select[0].selected, false);
-          assert.equal($terms1Select[0].selected, false);
+          var $allSelects = $node.find("#vocabularies select");
+          assert.equal($allSelects.size(), 2);
+
+          var $vocabSelect = $node.find($allSelects).first();
+          assert.equal($vocabSelect.size(), 1);
+
+          // first vocab, two options
+          var $vocabOptions = $vocabSelect.find("option");
+          assert.equal($vocabOptions.size(), 2);
+
+          assert.equal($vocabOptions[0].selected, true);
+          assert.equal($vocabOptions[1].selected, false);
 
           // TestUtils.Simulate.change only simulates a change event,
           // we need to update the value first ourselves
-          $($vocabSelect[0]).val("hard").trigger('change');
-          React.addons.TestUtils.Simulate.change($vocabSelect[0]);
+          $vocabSelect.val("prerequisite").trigger('change');
+          React.addons.TestUtils.Simulate.change($vocabSelect);
```

Comment:
It looks like you missed this here. The change event may be provided to React via `trigger()` in which case you can just remove the `Simulate.change` line.